lib/ProductOpener/Food.pm

```diff
@@ -425,7 +425,7 @@ It is a list of nutrients names with eventual prefixes and suffixes:
 		(
 			'!energy-kj', '!energy-kcal',
 			'!energy-', '-energy-from-fat-',
-			'!fat', '!-saturated-fat',
```

Comment:
We want to keep the ! in !-saturated-fat, it indicates that when we display the nutrition facts table in the product page (not the product edit form), we want to display Saturated fat even if we don't have a value for the product.